Dockerfile

```diff
@@ -1,32 +1,65 @@
-FROM node:0.12
+# Build:
+# docker build -t lirantal/meanjs .
+#
+# Run:
+# docker run -it lirantal/meanjs
+
+FROM ubuntu:latest
+MAINTAINER Liran Tal <liran.tal@gmail.com>
+
+# Install Utilities
+RUN apt-get update -q
+RUN apt-get install -yqq wget aptitude htop vim vim-puppet git traceroute dnsutils curl ssh sudo psmisc gcc make build-essential libfreetype6 libfontconfig tree tcpdump nano
```

Comment:
understood, yet the idea is to provide a development environment, not propose an ideal docker setup for production.
are there any packages you think should be strictly left out?